### ISSUE TITLE
[Benchmarks] Deprecate `import intel_extension_for_pytorch` in benchmarks including decoupling XeTLA build which require `ipex.h`

### ISSUE THREAD

**Issue by vlad-penkin** [intel/intel-xpu-backend-for-triton#1911]

**Comment by ESI-SYD** [comment]
Status: 

We have changed the codes and have an initial performance impact evaluation. There are 3 points need to be addressed for now.

1. Upstream pytorch attention as golden reference for triton not works which is a blocker https://github.com/intel/intel-xpu-backend-for-triton/issues/2042.
2. Ratio of `Triton`/ `XeTLA` keep same except for attention casued by `XeTLA` attention degraded
3. Both `Triton` and `XeTLA` softmax cases degraded, so `Triton`/ `XeTLA` not changed.



details: https://github.com/intel/intel-xpu-backend-for-triton/pull/1905#issuecomment-2320701513

**Comment by vlad-penkin** [comment]
@ESI-SYD please create a new ticket for items 2 and 3.

**Comment by ESI-SYD** [comment]
https://github.com/intel/intel-xpu-backend-for-triton/issues/2106
